examples/puzzles/tictactoe/tictactoe.qnt

```diff
@@ -1,29 +1,16 @@
 // -*- mode: Bluespec; -*-
+/**
+ * A simple definition of the tictactoe game based on [1]. Player X will
+ * always play the best move while Player O will play a random valid move.
+ *
+ * [1]: https://elliotswart.github.io/pragmaticformalmodeling/
+ *
+ * Gabriela Moreira, Informal Systems, 2022-2023
+ */
 module tictactoe {
   var board: int -> (int -> str) // board[1..3][1..3] A 3x3 tic-tac-toe board
```

Comment:
That actually makes more sense and it's easier to read. Unfortunately, the traces stop looking like a tictactoe board :disappointed: 

before:
![image](https://github.com/informalsystems/quint/assets/18356186/ec70115b-6d3b-43b8-b181-bbe40d58f38c)

after: 
![image](https://github.com/informalsystems/quint/assets/18356186/e77388a5-2b3a-4366-beb2-aad68494b252)

Reply:
@konnov WDYT? Change it or leave the trace-friendly option?

Reply:
That's interesting. Actually, why don't we use lists here? :)

Reply:
I just tried. Lists make the code the worst (in the few defs that manipulate the board), and the traces are still not as nicely formatted.
![image](https://github.com/informalsystems/quint/assets/18356186/4f329d47-3f14-48ed-b0ec-82272d6f4475)

Reply:
This might be a good kind of usecase for "views"/"representation", as suggested in https://github.com/informalsystems/quint/issues/853#issuecomment-1585151748

Reply:
Yes! We should have a way to control this.

Reply:
I will just keep the maps for now and we can revisit to play around with custom views when we have them.